internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -114,25 +114,27 @@ func TestColumn_Header(t *testing.T) {
 func TestColumn_Template_Body(t *testing.T) {
 	t.Parallel()
 
-	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "body.key1"}
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "Body('key1.key2')"}
```

Comment:
Niekde by mal byt test aj na to, ked je dana hodnota object/array, ... teda `Body('key1')`

Reply:
https://github.com/keboola/keboola-as-code/pull/1031/commits/84f21dbd2be7288bc392e424a567fd1de806f31f